lib/ZumoOta/src/FileManager.cpp

```diff
@@ -55,7 +58,8 @@
 /******************************************************************************
  * Local Variables
  *****************************************************************************/
-
+/** Serial log sink */
+static LogSinkPrinter gLogSinkSerial("Serial", &Serial);

```

Comment:
I doubt you need this here again as logging is a singleton. Only the one in App should be enough.

Reply:
It will be taken care of by the next commit.